File: microservices/ingrediente/src/main/java/br/edu/infnet/ReceitaFacil/ingrediente/model/Item.java
package br.edu.infnet.ReceitaFacil.ingrediente.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter@Setter
@NoArgsConstructor@AllArgsConstructor
public class Item {
    private Long id;
    private String descricao;
    private float preco;
    private Long medidaId;
}

File: android/app/src/main/java/br/edu/infnet/receitafacil_microservices/model/Item.kt
package br.edu.infnet.receitafacil_microservices.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Item (
    var id: Long,
    var descricao: String,
    var preco: Float,
    var medida: Medida
): Parcelable